--- musikernel-master/src/pydaw/Makefile.orig   2015-05-24 14:52:17.758474200 -0700
+++ musikernel-master/src/pydaw/Makefile        2015-05-24 13:59:13.000000000 -0700
@@ -28,9 +28,8 @@

 PLAT_LINK_CFLAGS ?= $(RT_LIB_FLAG) $(ALSA_FLAG) $(LIBLO_FLAG)

-LINK_CFLAGS  = -lpthread $(PORTMIDI_FLAG) -lportaudio $(LDFLAGS) \
-$(shell pkg-config --libs sndfile fftw3f) $(MATH_LIB_FLAG) \
-$(PLAT_LINK_CFLAGS)
+LINK_CFLAGS  = -lpthread -lportaudio $(LDFLAGS) $(PORTMIDI_FLAG) \
+$(shell pkg-config --libs sndfile fftw3f) $(MATH_LIB_FLAG)

 C_OBJS   = src/main.o

